test/client_options_test.py

```diff
@@ -139,11 +140,17 @@ def config(tmpdir, monkeypatch):
 #username is not set
 password = password
 
-[invalid]
+[invalid1]
 engine_url = https://engine.com
 username = username
 password = password
 log_level = invalid log level
+
+[invalid2]
+engine_url = https://engine.com
+username = username
+password = password
+secure = invalid value
```

Comment:
for consistency with the other invalid config:

    secure = invalid secure

Or just use 'invalid value' in both to simplify. This will also make the test simpler, we will not need the value argument since the value is always the same.